README.md

```diff
@@ -60,6 +60,33 @@ class ExampleTest < ActionDispatch::IntegrationTest
 end
 ```
 
+If you want to test the broadcasting made with `Channel.broadcast_to`, you shoud use
+`Channel.broadcasting_name_for`\* to generate an underlying stream name:
+
+```ruby
+# app/jobs/chat_relay_job.rb
+class ChatRelayJob < ApplicationJob
+  def perform_later(room, message)
+    ChatChannel.broadcast_to room, text: message
+  end
+end
+
+class ChatRelayJobTest < ActiveJob::TestCase
+  include ActionCable::TestHelper
+
+  test "broadcast message to room" do
+    room = rooms(:all)
+
+    assert_broadcast_on(ChatChannel.broadcasting_name_for(room), text: "Hi!") do
```

Comment:
@sponomarev WDYT?

Reply:
> Honestly, I didn't really like to introduce such kind of incompatibility between Rails 5 and 6, but I didn't see a better solution of the top of my head.

I have one idea, which could provide a fully compatible API: Refinements.

We can refine `broadcasting_for` and use it like we're in Rails 6.

The only requirement is to add `using ActionCable::Testing::Syntax` to your test/spec file.

Reply:
that sounds great

Reply:
Ok, will do